contributors/devel/kubectl-conventions.md

```diff
@@ -266,7 +267,7 @@ rather than "RESOURCE" or "KIND"
 
 * Use "NAME" for resource names
 
-## kubectl Factory conventions
+## Kubectl Factory conventions
```

Comment:
This is unnecessary, it's acceptable to use `kubectl` lower case at the beginning of a title when referring to a CLI command.